OpenKeychain/src/main/java/org/sufficientlysecure/keychain/javacard/JavacardDevice.java

```diff
@@ -0,0 +1,105 @@
+package org.sufficientlysecure.keychain.javacard;
+
+import org.sufficientlysecure.keychain.pgp.CanonicalizedSecretKey;
+import org.sufficientlysecure.keychain.util.Passphrase;
+
+import java.io.IOException;
+
+public interface JavacardDevice {
```

Comment:
why the abstraction, do you expect to have other implementations here? did you want to split this up some way, apart from the transport-abstraction?